tools/create_f2fs_image.sh

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
+SYZDIR=/usr/local/google/home/jiahengh/gopath/src/github.com/google/syzkaller
+lab1="-a 1 "
+lab2="-s 2 "
+lab3="-m "
+declare -i dex=0
+
+fallocate -l 64M disk.raw && mkfs.f2fs disk.raw 
+go run "$SYZDIR/tools/syz-imagegen/imagegen.go" -image=./disk.raw -fs=f2fs > syz_image_mount_f2fs_$dex
+rm disk.raw
+dex=dex+1
+
+fallocate -l 128M disk.raw && mkfs.f2fs disk.raw 
+go run "$SYZDIR/tools/syz-imagegen/imagegen.go" -image=./disk.raw -fs=f2fs > syz_image_mount_f2fs_$dex
+rm disk.raw
+dex=dex+1
+
+fallocate -l 64M disk.raw && mkfs.f2fs $lab1 disk.raw 
+go run "$SYZDIR/tools/syz-imagegen/imagegen.go" -image=./disk.raw -fs=f2fs > syz_image_mount_f2fs_$dex
+rm disk.raw
+dex=dex+1
+
+fallocate -l 64M disk.raw && mkfs.f2fs $lab2 disk.raw 
+go run "$SYZDIR/tools/syz-imagegen/imagegen.go" -image=./disk.raw -fs=f2fs > syz_image_mount_f2fs_$dex
+rm disk.raw
+dex=dex+1
+
+fallocate -l 64M disk.raw && mkfs.f2fs $lab3 disk.raw 
+go run "$SYZDIR/tools/syz-imagegen/imagegen.go" -image=./disk.raw -fs=f2fs > syz_image_mount_f2fs_$dex
+rm disk.raw
+dex=dex+1
+
+fallocate -l 64M disk.raw && mkfs.f2fs $lab2 $lab3 disk.raw 
+go run "$SYZDIR/tools/syz-imagegen/imagegen.go" -image=./disk.raw -fs=f2fs > syz_image_mount_f2fs_$dex
+rm disk.raw
+dex=dex+1
+
+fallocate -l 64M disk.raw && mkfs.f2fs $lab2 $lab1 disk.raw 
+go run "$SYZDIR/tools/syz-imagegen/imagegen.go" -image=./disk.raw -fs=f2fs > syz_image_mount_f2fs_$dex
+rm disk.raw
+dex=dex+1
+
+fallocate -l 64M disk.raw && mkfs.f2fs $lab1 $lab3 disk.raw 
+go run "$SYZDIR/tools/syz-imagegen/imagegen.go" -image=./disk.raw -fs=f2fs > syz_image_mount_f2fs_$dex
+rm disk.raw
+dex=dex+1
+
+fallocate -l 64M disk.raw && mkfs.f2fs $lab1 $lab2 $lab3 disk.raw 
+go run "$SYZDIR/tools/syz-imagegen/imagegen.go" -image=./disk.raw -fs=f2fs > syz_image_mount_f2fs_$dex
+rm disk.raw
+dex=dex+1
```

Comment:
Too much repetition.  Can we use some for loops?
e.g,

```
local flags=("" "-a 1" "-s 2 " "-m")
local -i index=1
for flag in "${flags[@]}"; do
  fallocate -l 64M disk.raw
  mkfs.f2fs $flag disk.raw
  go run "$SYZDIR/tools/syz-imagegen/imagegen.go" -image=./disk.raw -fs=f2fs > syz_image_mount_f2fs_$index
  rm disk.raw
  (( index += 1 ))
done
```

Reply:
Also if you need multiple loops, it may also be worth factoring out the shared logic into a helper function.